.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
Do we need this file?

Reply:
nope. adding to .gitignore.

Reply:
If we don't need it, can we just delete the file?

Reply:
should be ignored now...

Reply:
It's ignored but still in git. `git rm .vscode/settings.json`